fuse_optimizers/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ catkin_package(
 ###########
 ## Build ##
 ###########
-add_compile_options(-Wall -Werror)
+add_compile_options(-Wall)  # -Werror)
```

Comment:
I haven't tried restoring Werror here since rebasing these commits on latest, maybe it's not needed now?